docker/nginx.conf

```diff
@@ -0,0 +1,16 @@
+server {
+  listen 80;
+
+  root /home/app/wikirate/public;
+
+  passenger_enabled on;
+  passenger_user app;
+
+  passenger_ruby /usr/bin/ruby3.3;
```

Comment:
I'll switch semaphore testing to ruby 3.3. They have 3.3.0. Probably close enough.